xwiki-platform-core/xwiki-platform-model/xwiki-platform-model-api/src/main/java/org/xwiki/model/internal/reference/AbstractEntityReferenceResolver.java

```diff
@@ -40,7 +40,10 @@
  */
 public abstract class AbstractEntityReferenceResolver
 {
-    protected static final Map<EntityType, EntityType> TYPE_CONVERT_MAPPING = new EnumMap<>(EntityType.class);
+    /**
+     * The mapping between the PAGE and the DOCUMENT based worlds.
+     */
+    public static final Map<EntityType, EntityType> TYPE_CONVERT_MAPPING = new EnumMap<>(EntityType.class);
```

Comment:
The name is not very explicit, couldn't it be named `PAGE_TYPE_CONVERT_MAPPING` or something like that?